src/app/datasets/[id]/DistributionAccordion.tsx

```diff
@@ -42,11 +43,14 @@ const DistributionAccordion = ({
   return (
     <div className="accordion flex w-full flex-col items-center justify-center">
```

Comment:
important: please add the same space you have between description and metadata, in between metadata and distributions.

The content is too packed right now.

<img width="1001" alt="Screenshot 2024-07-30 at 15 21 09" src="https://github.com/user-attachments/assets/8a3d20fe-f222-4585-a0e7-a8870ed2841f">

Reply:
done